### PR TITLE
fix(category option combo): Crash on event initial

### PR DESCRIPTION
**Pull request by Balcan**
Jira issues:
[ANDROAPP-2497](https://jira.dhis2.org/browse/ANDROAPP-2497)
[ANDROAPP-2501](https://jira.dhis2.org/browse/ANDROAPP-2501)

### CODE REVIEW

**Comment by JaimeToca** [comment]
@Balcan Can you reformatt the class to the normal java code style please ? It's very hard to follow what have change in eventInitialPresenter. Also we'll get conflicts when merging...

File: app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventInitial/EventInitialPresenter.java [hunk 1]
```diff
@@ -37,451 +34,471 @@
...
-                attrOptionComb = catOptComb.uid();
-        return attrOptionComb;
+    public String getCatOptionCombo( List<CategoryOptionCombo> categoryOptionCombos, List<CategoryOption> values )
+    {
+        return eventInitialRepository.getCategoryOptionCombo( catCombo.uid(), UidsHelper.getUidsList( values ) );
+        /*
+         * String attrOptionComb = ""; for (CategoryOptionCombo catOptComb :
+         * categoryOptionCombos) if (catOptComb.categoryOptions().containsAll(values))
+         * attrOptionComb = catOptComb.uid(); return attrOptionComb;
+         */
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/eventsWithoutRegistration/eventInitial/EventInitialPresenter.java]: do we need this? can u remove it?

**Comment by ferdyrod** [comment]
This is already tested on device!
